lib/wpt/templates/README.md

```diff
@@ -1,9 +1,16 @@
 # Web Platform Test Fixtures
 
-The fixtures here are pulled from https://github.com/web-platform-tests/wpt
-using [`git node wpt`](https://github.com/nodejs/node-core-utils/blob/master/docs/git-node.md#git-node-wpt)
+The files in this folder, including this document,
+are generated by [`git node wpt`][].
 
+This folder contains a subset of the [Web Platform Test][] for the
+implementation of Web APIs in Node.js.
+
+See [test/wpt](../../wpt/README.md) on how these tests are run.
```

Comment:
Currently 404?

Reply:
@vsemozhetbyt Yes, this is what it looks like eventually: https://github.com/joyeecheung/node/tree/wpt-any/test/wpt so it will not be a 404 once the file this tool generates gets merged in core with this branch.